src/Cli.Tests/UpdateRuntimeTests.cs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Cli.Tests
+{
+    /// <summary>
+    /// Tests to validate cli commands to update runtime settings.
+    /// </summary>
+    [TestClass]
+    public class UpdateRuntimeTests : VerifyBase
+    {
+        private MockFileSystem? _fileSystem;
+        private FileSystemRuntimeConfigLoader? _runtimeConfigLoader;
+        private static string _testConfigFile = "test-update-runtime-setting.json";
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            _fileSystem = FileSystemUtils.ProvisionMockFileSystem();
+
+            _runtimeConfigLoader = new FileSystemRuntimeConfigLoader(_fileSystem);
+
+            ILoggerFactory loggerFactory = TestLoggerSupport.ProvisionLoggerFactory();
+
+            SetLoggerForCliConfigGenerator(loggerFactory.CreateLogger<ConfigGenerator>());
+            SetCliUtilsLogger(loggerFactory.CreateLogger<Utils>());
+        }
+
+        /// <summary>
+        /// Test to validate when update runtime setting command is used with no options
+        /// no changes are made to the config file.
+        /// </summary>
+        [TestMethod]
+        public void TestNoUpdateRuntimeSettings()
+        {
+            _fileSystem!.AddFile(_testConfigFile, new MockFileData(INITIAL_CONFIG));
+
+            UpdateRuntimeOptions options = new(
+                depthLimit: null,
+                config: _testConfigFile
+            );
+
+            Assert.IsTrue(_fileSystem!.File.Exists(_testConfigFile));
+            Assert.IsTrue(TryUpdateRuntimeSettings(options, _runtimeConfigLoader!, _fileSystem!));
+
+            string updatedConfig = _fileSystem!.File.ReadAllText(_testConfigFile);
+            // Assert that INITIAL_CONFIG is same as the updated config
+            Assert.IsTrue(JToken.DeepEquals(JObject.Parse(INITIAL_CONFIG), JObject.Parse(updatedConfig)));
+        }
+
+        /// <summary>
+        /// Test to validate when no limit was specified, it gets added to the config file
+        /// using update runtime command. Here we are adding depth limit of 8 for GraphQL.
+        /// Once added it should be present in the config file.
+        [TestMethod]
```

Comment:
this summary is worded weird. "when no limit was specified," not specified when? when running `dab runtime --query-depth` ?

perhaps a clearer summary would state that: running "dab runtime --query-depth 8" on a config with no depth previously specified, results
in runtime.graphql.querydepthlimit property being set with the value 8.